python/lsst/ap/association/ssoAssociation.py

```diff
@@ -125,12 +125,11 @@ def run(self, diaSourceCatalog, solarSystemObjects, exposure):
         maskedObjects = self._maskToCcdRegion(
             solarSystemObjects,
             exposure,
-            solarSystemObjects["Err(arcsec)"].max())
+            solarSystemObjects["Err(arcsec)"].max()).copy()
         nSolarSystemObjects = len(maskedObjects)
         if nSolarSystemObjects <= 0:
-            return self._return_empty(diaSourceCatalog)
+            return self._return_empty(diaSourceCatalog, maskedObjects)
 
-        self.log.info("Attempting to associate %d objects...", nSolarSystemObjects)
```

Comment:
Why did you delete this log message?

Reply:
Amalgamated into "Successfully associated %d / %d SolarSystemObjects.", which I prefer since multithreaded log messages shuffle the lines together.